### PR TITLE
Temporarily disable IBC merge.

### PR DESCRIPTION
**Pull request by tannergooding**
The issue is being tracked here: https://github.com/dotnet/roslyn/issues/12508

FYI. @KevinH-MS, @jasonmalinowski, @dotnet/roslyn-infrastructure

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @CyrusNajmabadi, @Pilchie

File: build/Targets/VSL.Imports.targets [hunk 1]
```diff
@@ -392,8 +392,11 @@
           Inputs="@(IntermediateAssembly)"
           Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
 
-    <Message Text="Adding optimization data to @(IntermediateAssembly)"/>
-    <Exec Command="&quot;$(IbcMergePath)&quot; -q -f -partialNGEN -minify -mo &quot;@(IntermediateAssembly)&quot; -incremental &quot;$(OptimizationDataFile)&quot;" />
+    
+    <!-- SKIPPING: https://github.com/dotnet/roslyn/issues/12508
+      <Message Text="Adding optimization data to @(IntermediateAssembly)"/>
+      <Exec Command="&quot;$(IbcMergePath)&quot; -q -f -partialNGEN -minify -mo &quot;@(IntermediateAssembly)&quot; -incremental &quot;$(OptimizationDataFile)&quot;" />
+    -->
```
Comment (Pilchie) [inline build/Targets/VSL.Imports.targets]: I suspect we won't be able to insert until this is dealt with, but :+1: to unblock PRs.

Comment (tannergooding) [inline build/Targets/VSL.Imports.targets]: I expect the same, I have marked the bug with `Urgency Now`

**Comment by KevinH-MS** [comment]
@tannergooding do we have a repro machine?

**Comment by tannergooding** [comment]
@KevinH-MS, any of the integration machines with this PR enabled. I'll create a PR reverting this change and take a machine explicitly offline after it fails.

**Pull request merged**